src/js/view/controls/controlbar.js

```diff
@@ -170,6 +170,9 @@ export default class Controlbar {
             cast: createCastButton(() => {
                 _api.castToggle();
             }, localization),
+            imaFullscreen: button('jw-icon-fullscreen', () => {
```

Comment:
Nice!

Reply:
If this is the fullscreen icon for all ads do we want to call it something like `adsFullscreen`?

Reply:
good call. It was originally just for IMA and then I realized the shared UI. Forgot to update the name.